### PR TITLE
DM-17095: Speed up tests

### PR DESCRIPTION
**Pull request by timj**
Creating a butler seems to take "forever" and we are doing it every test. This PR creates a butler ever test class. Tests now take 3 minutes rather than 15 minutes on my laptop.

### CODE REVIEW

File: tests/test_auxTel.py [hunk 1]
```diff
@@ -24,28 +24,15 @@
 import unittest
 
 import lsst.utils.tests
-from lsst.utils import getPackageDir
 from lsst.afw.geom import arcseconds, Extent2I
 import lsst.afw.image
 import lsst.obs.base.tests
 
+from obsLsstTestHelper import ObsLsstButlerTests
```
Comment (SimonKrughoff) [inline tests/test_auxTel.py]: Should this be explicitly relative?

File: tests/test_auxTel.py [hunk 2]
```diff
@@ -24,28 +24,15 @@
...
 
-class TestAuxTel(lsst.obs.base.tests.ObsTests, lsst.utils.tests.TestCase):
```
Comment (SimonKrughoff) [inline tests/test_auxTel.py]: There is something wrong with this diff.  The PR is asking to go to master, but these class level tearDown and setUp do not exist on master?  Was it branched from another ticket, and if so, will that ticket get merged first?

Comment (SimonKrughoff) [reply tests/test_auxTel.py]: Oh never mind.  I see that you did it in two pieces.  I think it would be clearer if these two commits were squashed.

Comment (timj) [reply tests/test_auxTel.py]: Well, one commit was the speed up commit. The second commit is the refactoring to pull the code into a helper class.

File: tests/test_auxTel.py (hunk 1 above)
Comment (timj) [reply tests/test_auxTel.py]: Only if we add a `__init__.py` I think.

File: tests/test_auxTel.py (hunk 2 above)
Comment (SimonKrughoff) [reply tests/test_auxTel.py]: I'm happy with either way.  It confused me because I read the commits in the wrong order.

**Pull request merged**